cookbooks/aws-parallelcluster-install/files/amazon/dkms/nvidia.conf

```diff
@@ -0,0 +1 @@
+MAKE[0]="'make' -j2 NV_EXCLUDE_BUILD_MODULES='' KERNEL_UNAME=${kernelver} modules CC=/usr/bin/gcc10-gcc"
```

Comment:
we need to be sure that `gcc10` is installed. If it's not already in the list of installed package, let's add it for this case (AL2 with Kernel 5)